### PR TITLE
FFWEB-1591 proofread ff-compare

### PR DESCRIPTION
**Pull request by omikron-fp**
Fixes typos, grammar and mustache code listing.

### CODE REVIEW

**Comment by omikron-fp** [comment]
> Shouldn't these changes also be added to version 1.x?

Alright, I'll add it.

**Comment by KrzysztofKozlowskiOBSS** [comment]
Weren't we supposed to abandon 1.x docs?

**Comment by wdutkiewicz-obj** [comment]
> Weren't we supposed to abandon 1.x docs?

Were we? I don't remember such arrangement. Moreover, as long as we have 1.x version users, we cannot terminate the 1.x docs... Just like IE11 - till the end of the world.

**Comment by omikron-fp** [comment]
Adapting these changes to 1.x was no effort so I quickly did it.
Discontinuation of 1.x docs remains debatable, however.

File: markdown/1.x/en/ff-compare.md [hunk 1]
```diff
@@ -19,135 +18,136 @@ This `ff-record-list` does not listen to the default search result and gets its
...
-If the compare service has no results for that given field, all **ff-compare-*** css will be removed.
+After the comparison, each field with a difference will add the CSS class `ffw-compare-diff` to the
+corresponding HTML Element.
+Or, when the fields are identical, it will add the `ffw-compare-equal` class.
+If the compare service has no results for that given field, all `ffw-compare-*` CSS classes will be removed.
 
 ```html
 <style>
-    .ff-compare-diff {
+    .ffw-compare-diff {
```
Comment (a-laurowski) [inline markdown/1.x/en/ff-compare.md]: Doesnt CSS classes starts with `ff-` in 1.x ?

**Comment by KrzysztofKozlowskiOBSS** [comment]
> > Weren't we supposed to abandon 1.x docs?
> 
> Were we? I don't remember such arrangement. Moreover, as long as we have 1.x version users, we cannot terminate the 1.x docs... Just like IE11 - till the end of the world.

I'm not talking about termination, but I'm pretty sure we agreed not to update them anymore.

**Pull request merged**